packages/dev/docs/src/docs.css

```diff
@@ -71,6 +71,9 @@
 html, body {
   margin: 0;
   text-size-adjust: none;
+  height: 100vh;
```

Comment:
This breaks the mobile layout. 100vh in iOS safari includes the top and bottom toolbars.